tests/test_aggregation.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from pyot import schema, aggregate
+
+
+# Const
+file_location = './tests/data/aggregation'
+spark_partition_num = 1  # Set 1 because local spark doesn't have much resource
+window_sec = 5  # Time window
+termination_time = 10
+stream_name = "input_stream"
+
+
+@pytest.fixture
+def spark():
+    from pyspark.sql import SparkSession
+    spark = SparkSession.builder.appName('StructredStreamingApp').getOrCreate()
+    return spark
+
+
+def test_aggregate_sensordata(spark):
+    # Write data in the console
+    initialDF = (spark
+                 .readStream
+                 # Read 1 file/trigger. It emmulcate streaming
```

Comment:
emmulcate? Do you mean accumulate?

Reply:
Sorry. This is typo. I want to say "emulate". Fixed comment.